packages/client-core/src/admin/components/Avatars/Avatars.tsx

```diff
@@ -179,11 +179,11 @@ const Avatars = (props: Props) => {
   // }, [])
 
   useEffect(() => {
-    if (user?.id.value != null && (adminAvatarState.avatars.updateNeeded.value === true || refetch === true)) {
+    if (user?.id.value != null && (adminAvatarState.updateNeeded.value || refetch === true)) {
       AvatarService.fetchAdminAvatars()
     }
     setRefetch(false)
-  }, [authState.user?.id?.value, adminAvatarState.avatars.updateNeeded.value, refetch])
+  }, [authState.user?.id?.value, adminAvatarState.updateNeeded, refetch])
```

Comment:
`adminAvatarState.updateNeeded` should probably be `adminAvatarState.updateNeeded.value`

Reply:
Fixed